.github/workflows/Operator-CI.yaml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: Build Shard operator image and resources
         uses: ./.github/actions/maven
         with:
-          maven-command: clean install --errors --quiet -Pminikube -Dquarkus.container-image.tag=$GITHUB_SHA -Dnamespace=$NAMESPACE -Dquickly
+          maven-command: clean install --errors --quiet -Pminikube -Dquarkus.container-image.tag=$GITHUB_SHA -Dnamespace=$NAMESPACE -Dquickly -Dquarkus.kubernetes.namespace=$NAMESPACE
```

Comment:
Is there any reason why we have now the `$NAMESPACE` property in two different places? As far as I see `-Dnamespace` is a common property that works both in k8s and openshift

Reply:
`-Dquarkus.kubernetes.namespace` is common property k8s and openshift. I have removed `-Dnamespace`.